src/uu/ls/src/ls.rs

```diff
@@ -52,6 +52,7 @@ use unicode_width::UnicodeWidthStr;
 use uucore::libc::{dev_t, major, minor};
 #[cfg(unix)]
 use uucore::libc::{S_IXGRP, S_IXOTH, S_IXUSR};
+#[cfg(any(not(unix), target_os = "android", target_os = "macos"))]
```

Comment:
This `cfg` seems unnecessary if you remove `line_ending::LineEnding` from line 67